### ISSUE TITLE
DRF 3.11+ compatibility - base_name renamed to basename in router.register

### ISSUE THREAD

**Issue by cezio** [yourlabs/django-cities-light#204]
With DRF 3.11 release, `base_name` param in `router.register` has been dropped in favor of `basename` (deprecated warning was emitted from DRF 3.9), which causes following error:

```
  File "cities_light/contrib/restframework3.py", line 117, in <module>
    router.register(r'cities', CityModelViewSet, base_name='cities-light-api-city')
TypeError: register() got an unexpected keyword argument 'base_name'
```

The fix is to use `basename` instead.

Related DRF issue: https://github.com/encode/django-rest-framework/pull/5990

**Comment by jpic** [comment]
Nice ! Did you have the chance to hack DCL already ? Can you post your working patch ?

Best

**Comment by Hardeepsingh980** [comment]
Same issue can you please fix and publish as soon as possible.

**Comment by jpic** [comment]
Released in 3.5.1, please confirm resolution

**Comment by jpic** [comment]
@bomba1990 agreed, transfered the repo to jazzband

https://github.com/jazzband/django-cities-light
